### ISSUE TITLE
better flask example

### ISSUE THREAD

**Issue by edublancas** [ploomber/doc#14]
Our current flask example is a hello world: https://github.com/ploomber/doc/tree/main/examples/docker/flask

we need to add more interesting examples.

one can be show how to train a model locally and then serve it via an API but open to other ideas

**Comment by neelasha23** [comment]
I have looked through some examples and the one I decided on is the Sales forecasting . The dataset I have found till now is the Walmart Sales (Kaggle dataset) but I'm not sure if we should use it since we decided to avoid Kaggle datasets. 
Reference code: https://jovian.com/sergioalves94/walmart-store-sales-forecasting
There are also some interesting dashboards related to this problem here:
 https://neptune.ai/resources/time-series-forecasting-example

I think we can include similar plots and charts. One webpage can be used to display training loss by epochs etc. We can have some more pages for each model and the forecasting trend plots similar to the one in the above link.

If we need to demonstrate something more complex we can also try computer vision projects/ LLM based projects.

Let me know what you think @edublancas
